docs/applications.rst

```diff
@@ -20,3 +20,18 @@ Streamlink Twitch GUI
 :Info: A NW.js based desktop web application, formerly known as *Livestreamer Twitch GUI*. Browse Twitch.tv and watch
     multiple streams at once. Filter streams by language, receive desktop notifications when followed channels start
     streaming and access the Twitch chat by using customizable chat applications.
+
+
+LiveStreamLinkGUI
+---------------------
+
+.. image:: https://raw.githubusercontent.com/techmouse/livestreamlinkgui/master/screenshot.png
+    :alt: LiveStreamLinkGUI
+
+:Description: A lightweight BASH script
+:Type: Graphical User Interface
+:OS: Linux and any other OS that supports BASH, Zenity, and typical *nix commands
```

Comment:
The `*` is causing this to fail on the doc build in Travis. Should be okay if you remove that or change it to `\*`.